src/dmd/mtype.h

```diff
@@ -94,7 +95,6 @@ enum ENUMTY
     TTraits,
     TMAX
 };
-typedef unsigned char TY;       // ENUMTY
 
 extern int Tsize_t;
```

Comment:
Shouldn't these be adjusted also? As you'll now potentially run into signed vs unsigned comparison warnings/bugs.

Reply:
I don’t. I don’t see it in `TY`/`ENUMTY`.

Reply:
github.com/dlang/dmd/pull/9960